docs/pt/docs/help-fastapi.md

```diff
@@ -0,0 +1,106 @@
+    * Fazer perguntas.
+* <a href="https://www.linkedin.com/in/tiangolo/" class="external-link" target="_blank">conecte-se comigo no **Linkedin**</a>.
+    * Falar comigo.
+    * Me recomende ou me endosse :)
+* <a href="https://medium.com/@tiangolo" class="external-link" target="_blank">Ler o que escrevo (ou me seguir) no **Medium**</a>.
+    * Ler outras ideias, artigos e ferramentas que eu criei.
+    * Me seguir para ver quando eu publico algo novo.
+
+## Tweet sobre **FastAPI**
+
+<a href="https://twitter.com/compose/tweet?text=Estou amando o FastAPI porque... https://github.com/tiangolo/fastapi cc @tiangolo" class="external-link" target="_blank">Tweet sobre **FastAPI**</a> e deixe eu e outros saberem porque você gosta do **FastAPI**.
+
+## Me fale como está usando o **FastAPI**
+
+Eu adoraria saber como o **FastAPI** está sendo usado, o que você gostou dele, e em qual projeto/empresa está sendo utilizado etc.
+
+Você pode me falar:
+
+* <a href="https://twitter.com/compose/tweet?text=Oi @tiangolo, Estou usando o FastAPI na..." class="external-link" target="_blank">No **Twitter**</a>.
+* <a href="https://www.linkedin.com/in/tiangolo/" class="external-link" target="_blank">No **Linkedin**</a>.
+* <a href="https://medium.com/@tiangolo" class="external-link" target="_blank">No **Medium**</a>.
+
+## Vote no FastAPI
+
+* <a href="https://github.com/vinta/awesome-python/pull/1209" class="external-link" target="_blank">Vote pare incluir o **FastAPI** no _`awesome-python`_</a>.
+* <a href="https://www.slant.co/options/34241/~fastapi-review" class="external-link" target="_blank">Vote pelo **FastAPI** no Slant</a>.
+
+## Ajude outros com as _issues_ no GitHub
+
+Você pode ver <a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">as _issues_ existentes</a> e tentar ajudar os outros.
+
+## Fique de olho nos lançamentos no repositório no GitHub
+
+Você pode "watch" o FastAPI no GitHub (clicando no botão escrito 'Watch' no canto superior direito): <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">https://github.com/tiangolo/fastapi</a>.
+
+Se você selecionar "Watching" ao invés de "Releases only" você receberá notificações quando alguém criar uma _issue_.
+
+Ai você pode tentar ajudar ele/ela a resolver suas _issues_.
+
+## Criar _issues_
+
+Você pode <a href="https://github.com/tiangolo/fastapi/issues/new/choose" class="external-link" target="_blank">criar uma nova _issue_</a> no repositório do GitHub, por exemplo para:
+
+* Reportar um _bug/issue_.
+* Sugerir um novo recurso.
+* Fazer uma pergunta.
+
+## Criar um _Pull Request_
+
+Você pode <a href="https://github.com/tiangolo/fastapi" class="external-link" target="_blank">criar um _Pull Request_</a> para, por exemplo:
+
+* Corrigir um erro na documentação.
+* Sugerir novas seções na documentação.
+* Corrigir um _bug/issue_ existente.
+* Adicionar um novo recurso.
+
```

Comment:
There are some sections of the original documentation missing here, I'm guessing they were added later?

- Join the chat (which now recommends Discord)
- Sponsor the author
- Sponsor the tools that power FastAPI

Would you consider adding those as well?

Other than that, excelent work!